examples/simple-flake/main.go

```
@@ -0,0 +1,232 @@
+	reflection.Register(s)
+
+	log.Printf("Server listening on %s", port)
+	if err := s.Serve(lis); err != nil {
+		log.Fatalf("failed to serve: %v", err)
+	}
+}
+
+// runClient demonstrates client usage
+func runClient(serverAddr string) {
+	// Create a connection to the server
+	conn, err := grpc.Dial(serverAddr, grpc.WithTransportCredentials(insecure.NewCredentials()))
```

golangci-lint (1.64.8)

SA1019: grpc.Dial is deprecated: use NewClient instead.  Will be supported throughout 1.x. 

(staticcheck)